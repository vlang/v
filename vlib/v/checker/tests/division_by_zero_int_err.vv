fn main() {
	println(1 / 0)
	println(1 / 0x0)
	println(1 / 0b0)
	println(1 / 0o0)
}
