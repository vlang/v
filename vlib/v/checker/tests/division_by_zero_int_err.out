vlib/v/checker/tests/division_by_zero_int_err.vv:2:14: error: division by zero
    1 | fn main() {
    2 |     println(1 / 0)
      |                 ^
    3 |     println(1 / 0x0)
    4 |     println(1 / 0b0)
vlib/v/checker/tests/division_by_zero_int_err.vv:3:14: error: division by zero
    1 | fn main() {
    2 |     println(1 / 0)
    3 |     println(1 / 0x0)
      |                 ~~~
    4 |     println(1 / 0b0)
    5 |     println(1 / 0o0)
vlib/v/checker/tests/division_by_zero_int_err.vv:4:14: error: division by zero
    2 |     println(1 / 0)
    3 |     println(1 / 0x0)
    4 |     println(1 / 0b0)
      |                 ~~~
    5 |     println(1 / 0o0)
    6 | }
vlib/v/checker/tests/division_by_zero_int_err.vv:5:14: error: division by zero
    3 |     println(1 / 0x0)
    4 |     println(1 / 0b0)
    5 |     println(1 / 0o0)
      |                 ~~~
    6 | }
